EIPS/eip-x.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: XXXX
+title: Add wallet_signIntendedValidatorData method
+description: A new RPC method to sign data with an intended validator address according to ERC-191 version 0x00.
+author: Yamen Merhi (@YamenMerhi)
+discussions-to: TBD
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR: https://ethereum-magicians.org/new